eng/versioning/version_client.txt

```diff
@@ -114,7 +114,7 @@ com.microsoft.azure:spring-data-cosmosdb;3.0.0-beta.1;3.0.0-beta.1
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
 unreleased_com.azure:azure-identity;1.1.0
-unreleased_com.azure:azure-messaging-servicebus;7.0.0-beta.4
+unreleased_com.azure:azure-messaging-servicebus;7.0.0-beta.5
```

Comment:
Who uses unreleased_servicebus?

Reply:
It's used down below in sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-servicebus/pom.xml

Reply:
used by sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-servicebus/pom.xml